spyder/plugins/projects/widgets/tests/test_projectdialog.py

```diff
@@ -9,10 +9,15 @@
 """
 
 # Test library imports
+import os
 import pytest
+try:
+    from unittest.mock import Mock
+except ImportError:
+    from mock import Mock  # Python 2
 
 # Local imports
-from spyder.plugins.projects.widgets.projectdialog import ProjectDialog
+from spyder.widgets.projects.projectdialog import ProjectDialog
```

Comment:
I'm pretty sure this had something to do with merging with ``master`` after ``split-plugins`` was merged, but yeah ``spyder.plugins.projects.widgets.projectdialog import ProjectDialog`` is now correct (``from spyder.widgets.projects.projectdialog import ProjectDialog`` was the previous path before ``split-plugins`` went in).

Reply:
I usually just ``rebase`` to keep the commit history nice and clean, and its only *really* necessary to merge or rebase with upstream if either there's a merge conflict, or you need some specific fix/functionality so the CIs pass or your code works, etc. (athough it doesn't hurt to rebase otherwise if you really want, since it doesn't pollute the history).

We could consider just ``squashing`` this on merge to ``master`` since the commit history is rather messy and uninformative with lots of duplicate and unclear messages.

Reply:
Yes, this change needs to be reverted.

@vitoriahmc, we just merged a huge refactoring, so all our plugins are now self-contained modules.